packages/@webex/webex-core/test/unit/spec/services-v2/interceptors/server-error.js

```
@@ -0,0 +1,204 @@
+        it('should return a rejected promise with the reason', (done) => {
+          interceptor.onResponseError(options, reason).catch((error) => {
+            assert.deepEqual(error, reason);
+
+            done();
+          });
+        });
+      });
+
+      describe('when the uri does not exist', () => {
+        beforeEach(() => {
+          delete options.uri;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)